docs/schema/cookieauthdata-complextype-sps15xsdsearchset2.md

```diff
@@ -1,26 +1,25 @@
 ---
 title: "CookieAuthData complexType (SPS15XSDSearchSet2)"
+description: Outlines information, a definition, and elements and attributes for the CookieAuthData complexType in Sharepoint.
 manager: arnek
-ms.date: 3/9/2015
+ms.date: 06/09/2022
 ms.audience: ITPro
 ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: 94086319-9184-17d6-2db7-c5a462e0a792
-description: "Last modified: March 09, 2015"
 ---
 
 # CookieAuthData complexType (SPS15XSDSearchSet2)
 
  
   
 ## Type information
+**Namespace**: http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration
```

Comment:
We've been directed to keep Markdown tables as tables.  You can use "Information/Location" or "Name/Value" as headers, or if you find a more specific header that fits, use that.